metadata/src/main/java/org/apache/kafka/controller/metrics/QuorumControllerMetrics.java

```diff
@@ -54,6 +54,14 @@ public class QuorumControllerMetrics implements AutoCloseable {
         "KafkaController", "LastAppliedRecordTimestamp");
     private final static MetricName LAST_APPLIED_RECORD_LAG_MS = getMetricName(
         "KafkaController", "LastAppliedRecordLagMs");
```

Comment:
I'd prefer it this way since we're initializing a bunch of constants here, and sometimes we want to grep for stuff.

It's kind of annoying when you have "matryoshka doll initialization" of constants.

Like rather than having:
```
static final String FOO = "foo";
static final String BAR = "bar";
static final String FOO_BAR = FOO + BAR;
static final String FOO_BAR_BAZ = FOO_BAR + "baz";
```

I'd rather just have:
```
static final String FOO = "foo";
static final String BAR = "bar";
static final String FOO_BAR = "foobar";
static final String FOO_BAR_BAZ = "foobarbaz";
```

since then I can grep for the constant and it works as expected.

Although maybe other people disagree?